### PR TITLE
Workaround to use STMCubeIDE in Windows 11

### PR DESCRIPTION
**Pull request by elgarbe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +93,21 @@ micro-ROS can be used with SMT32CubeIDE following these steps:
...
+
+1. Install Docker Desktop and open it
+2. Clone this repository in your STM32CubeIDE project folder
+3. Open a terminal in the project folder and run:
+   ```bash
+   docker pull microros/micro_ros_static_library_builder:humble
+   docker run --rm -v <ABSOLUTE_PATH_TO_PROJECT>:/project --env MICROROS_LIBRARY_FOLDER=micro_ros_stm32cubemx_utils/microros_static_library_ide microros/micro_ros_static_library_builder:humble
+   ```
+Follow steps 4 to 8.
+Noticed that thist steps where tested with ROS humble, but should work on any distribution. microROS agent could be build and run in Ubuntu using WSL.
```
Comment (pablogs9) [inline README.md]:
```suggestion
Noticed that thist steps where tested with ROS 2 Humble, but should work on any distribution. micro-ROS Agent could be build and run in Ubuntu using WSL.
```

File: README.md [hunk 2]
```diff
@@ -92,6 +93,21 @@ micro-ROS can be used with SMT32CubeIDE following these steps:
 6. Make sure that if you are using FreeRTOS, the micro-ROS task **has more than 10 kB of stack**: [Detail](.images/Set_freertos_stack.jpg)
 7. Configure the transport interface on the STM32CubeMX project, check the [Transport configuration](#Transport-configuration) section for instructions on the custom transports provided.
 8. Build and run your project
+
+### Windows 11
```
Comment (pablogs9) [inline README.md]:
```suggestion
### Windows 11 (Community Contributed)
```

File: README.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ This tool aims to ease the micro-ROS integration in a STM32CubeMX/IDE project.
   - [Middlewares available](#middlewares-available)
   - [Using this package with STM32CubeMX](#using-this-package-with-stm32cubemx)
   - [Using this package with STM32CubeIDE](#using-this-package-with-stm32cubeide)
+      - [Windows 11](#STM32CubeIDE-Win11)
```
Comment (pablogs9) [inline README.md]:
```suggestion
      - [Windows 11 (Community Contributed)](#STM32CubeIDE-Win11)
```

**Review by pablogs9 — COMMENTED** [review]
Thanks a lot for the contribution! Just some minor comments and we can merge.

**Comment by pablogs9** [comment]
@mergify backport humble rolling

**Comment by elgarbe** [comment]
I don't know how to squash all commits, mybe you can do it when you merge?

**Pull request merged**